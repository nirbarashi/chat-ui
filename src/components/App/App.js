
//This is your top level React component, you may change everything

import React from 'react'
// import logo from '../../assets/spotim-logo.jpg'
// import {Container, Image} from 'semantic-ui-react'
import Chat from '../Chat/Chat'


// class App extends React.PureComponent {
//   render() {
//     return <Container className={'spotim-header'}>
//       <div className={'spotim-title'}>
//         Welcome to the Spot.IM Chat app
//       </div>
//       <div>
//         <Logo>
//           <Image size={'tiny'} src={logo}/>
//         </Logo>
//
//       </div>
//     </Container>
//   }
// }

const App = () => {
	return (
	    <Chat/>
    )
}

export default App;