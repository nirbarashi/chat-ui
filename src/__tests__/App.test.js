import React from 'react';
import { create } from 'react-test-renderer';
import App from '../components/App/App';


describe("App", () => {
	test("Matches the snapshot", () => {
		const app = create(<App />);
		expect(app.toJSON()).toMatchSnapshot();
	});
});