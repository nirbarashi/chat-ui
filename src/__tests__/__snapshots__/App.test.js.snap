// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches the snapshot 1`] = `
<div
  className="chat"
>
  <header
    className="chat-header"
  >
    Welcome to the Spot.IM Chat app
  </header>
  <main
    className="chat-messages"
  >
    <ol
      className="chat-messages-list"
    >
      <li
        className="chat-message "
      >
        <header
          className="profile"
        >
          <img
            alt=""
            className="profile-image"
            src="003-pikachu.png"
          />
          <span
            className="profile-name"
          >
            Nir
          </span>
        </header>
        <p
          className="chat-message-content"
        >
          Welcome friend, feel free to send messages 
        </p>
      </li>
      <li
        className="chat-message "
      >
        <header
          className="profile"
        >
          <img
            alt=""
            className="profile-image"
            src="001-snorlax.png"
          />
          <span
            className="profile-name"
          >
            Alon22
          </span>
        </header>
        <p
          className="chat-message-content"
        >
          Thank you !
        </p>
      </li>
    </ol>
  </main>
  <div
    className="message-box"
  >
    <div
      className="message-box-user-info"
    >
      <img
        alt=""
        className="profile-image"
        src="003-pikachu.png"
      />
      <input
        className="message-box-input user-info-username"
        onChange={[Function]}
        placeholder="Enter your name here..."
        type="text"
        value=""
      />
    </div>
    <input
      className="message-box-input"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Type your message here..."
      type="text"
      value=""
    />
    <button
      className="message-box-button"
      onClick={[Function]}
    />
  </div>
</div>
`;
